<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlasmoidExecutionPage</class>
 <widget class="QWidget" name="PlasmoidExecutionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>616</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Plasmoid</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Path or name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="identifier">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Behavior</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Form Factor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="formFactor">
        <item>
         <property name="text">
          <string>planar</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>horizontal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>vertical</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mediacenter</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Theme</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="themes"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Dependencies</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Targets:</string>
        </property>
        <property name="buddy">
         <cstring>targetDependency</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="ProjectItemLineEdit" name="targetDependency">
          <property name="toolTip">
           <string>Enter a dependency to add to the list</string>
          </property>
          <property name="clickMessage">
           <string>Enter a dependency to add to the list</string>
          </property>
          <property name="showClearButton" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="browseProject"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="addDependency">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Adds the listed target to the dependency list.</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="4" colspan="2">
         <widget class="KListWidget" name="dependencies">
          <property name="toolTip">
           <string>List of indirect dependent targets.</string>
          </property>
          <property name="whatsThis">
           <string>This list should contain targets that the application does not directly depend on, but for which an action needs to be taken before running the application.</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="uniformItemSizes">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="removeDependency">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Removes the selected dependencies from the list.</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="moveDepUp">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Move a dependency up in the list.</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="moveDepDown">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Moves the selected dependency down in the list.</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>ProjectItemLineEdit</class>
   <extends>KLineEdit</extends>
   <header>project/projectitemlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
